test/functional/interface_zmq.py

```diff
@@ -490,12 +490,6 @@ def test_mempool_sync(self):
         mempool_view = set(mempool_snapshot["txids"])
         get_raw_seq = mempool_snapshot["mempool_sequence"]
         assert_equal(get_raw_seq, 6)
-        # Snapshot may be too old compared to zmq message we read off latest
-        while zmq_mem_seq >= get_raw_seq:
-            sleep(2)
-            mempool_snapshot = self.nodes[0].getrawmempool(mempool_sequence=True)
-            mempool_view = set(mempool_snapshot["txids"])
-            get_raw_seq = mempool_snapshot["mempool_sequence"]
```

Comment:
Would it make sense to assert that this is never true anymore?
```suggestion
        assert(zmq_mem_seq < get_raw_seq)
```

Reply:
Added as belt and suspender